### PR TITLE
Showing `Shortcuts.debugLabel` on its `Focus.debugLabel`

### PR DESCRIPTION
**Pull request by busslina**

### CODE REVIEW

File: packages/flutter/lib/src/widgets/shortcuts.dart [hunk 1]
```diff
@@ -1068,7 +1068,7 @@ class _ShortcutsState extends State<Shortcuts> {
   @override
   Widget build(BuildContext context) {
     return Focus(
-      debugLabel: '$Shortcuts',
+      debugLabel: '$Shortcuts${debugLabel == null ? '' : ' -- $debugLabel'}',
```
Comment (gspencergoog) [inline packages/flutter/lib/src/widgets/shortcuts.dart:1071]:
```suggestion
      debugLabel: '$Shortcuts${debugLabel?.isEmpty ?? true ? '' : ' -- $debugLabel'}',
```

To also print nothing when the debug label is the empty string.

**Review by gspencergoog — COMMENTED** [review]
Thanks for the PR! In order to continue with the review, we're going to need a test here.  Something that would have failed before your PR's changes, and passes after.

**Comment by gspencergoog** [comment]
You'll also need to add code for the case where the debug label was updated in the widget.  You'll need to set the debug label on the focus node whenever the widget's debug label changes.

**Comment by busslina** [comment]
I don't want to sign Google CLA, so I think I can't continue with this PR. But anyways, it would be helpful if any Dart member could add this functionality.

Thanks

**Pull request closed without merging**